src/services/dataset/DatasetService.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+class DatasetService {
+  constructor (apiService) {
+    this.apiService = apiService;
+    this.datasetApi = new apiService.DatasetApi();
+  }
+
+  async findDatasetById (organizationId, datasetId) {
+    const dataset = await this.datasetApi.findDatasetById(organizationId, datasetId);
+    return dataset;
```

Comment:
This syntax is equivalent to a "return await", which is not recommended (unless it is in a try/catch block).
We could simplify that into:
```
return this.datasetApi.findDatasetById(organizationId, datasetId);
```

The same could be done in other methods and  other services.

For more details about the "return await":

- https://jakearchibald.com/2017/await-vs-return-vs-return-await/
- https://eslint.org/docs/rules/no-return-await